README.md

```diff
@@ -338,6 +338,7 @@ Some of the supported options are:
 | `BLAS_ENABLE_STATIC_LIBRARY` | `ON`/`OFF` | Build as a static library (`OFF` by default) |
 | `ENABLE_EXPRESSION_TESTS` | `ON`/`OFF` | Build additional tests that use the header-only framework (e.g to test expression trees); `OFF` by default |
 | `BLAS_VERIFY_BENCHMARK` | `ON`/`OFF` | Verify the results of the benchmarks instead of only measuring the performance. See the documentation of the benchmarks for more details. `ON` by default |
+| `MODEL` | name | Pass a model name here to use optimized GEMM configurations for specific convolution models/sizes. Currently this only affects the `ARM_GPU` platform. The supported models are: `RESNET_50`, `VGG_16` |
```

Comment:
Not sure what the convention is, but options are often prefixed, so in this case it might be `BLAS_MODEL` or something like that.

Reply:
I think I was going for similar to ```TARGET``` but the convention is a bit inconsistent in this table! How about ```BLAS_MODEL_OPTIMIZATION``` a bit more verbose but I think it was a bit ambiguous before.